test/libsolidity/ASTJSON/constructor_legacy.json

```diff
@@ -52,6 +52,7 @@
             "scope": 5,
             "stateMutability": "nonpayable",
             "superFunction": null,
```

Comment:
I know its not part of this PR, but ugh, having `null`s in the JSON is weird (but technically valid JSON).

Reply:
you'd rather have no member `superFunction` at all in that case?

Reply:
Well we can discuss this outside of this PR. Can you open an issue tracking it?

Reply:
https://github.com/ethereum/solidity/issues/7897